app/views/contacts/_contact.html.erb

```diff
@@ -1,59 +1,57 @@
+      } %>
+    <% end %>
 
-    <div class="vcard contact-inner">
-      <% unless local_assigns[:hide_title] %>
-      <p><%= contact.title %></p>
-      <% end %>
+    <div class="govuk-grid-row">
       <% if contact.has_postal_address? %>
-        <%= render_hcard_address(contact) %>
+        <address class="govuk-grid-column-one-third contact-section__address">
+          <%= render_hcard_address(contact) %>
+        </address>
       <% end %>
 
       <% if contact.email.present? || contact.contact_form_url.present? || contact.contact_numbers.any? %>
-        <div class="email-url-number">
+        <div class="govuk-grid-column-one-third">
           <% if contact.email.present? %>
-            <p class="email">
-              <span class="type"><%= t('contact.email') %></span>
-              <%= mail_to contact.email, contact.email, class: "email govuk-link" %>
+            <p class="govuk-!-margin-bottom-4">
+              <%= t('contact.email') %><br />
+              <%= mail_to contact.email, contact.email, class: "govuk-link" %>
             </p>
           <% end %>
+
           <% if contact.contact_form_url.present? %>
-            <p class="contact_form_url">
-              <span class="type"><%= t('contact.contact_form') %></span>
+            <p class="govuk-!-margin-bottom-4">
+              <%= t('contact.contact_form') %><br />
               <%= link_to contact.contact_form_url.truncate(25), contact.contact_form_url, class: "govuk-link" %>
             </p>
           <% end %>
+
           <% contact.contact_numbers.each do |number| %>
-            <p class="tel">
-              <span class="type"><%= number.label %></span>
+            <p class="govuk-!-margin-bottom-4">
+              <%= number.label %> <br />
               <%= number.number %>
             </p>
           <% end %>
         </div>
       <% end %>
 
       <% if contact.comments.present? %>
-        <p class="comments">
+        <div class="govuk-grid-column-one-third">
           <%= auto_link(format_with_html_line_breaks(h(contact.comments)), html: { class: "govuk-link" }) %>
-        </p>
-      <% end %>
-
-      <% if contact.is_a?(WorldwideOffice) && contact.access_and_opening_times_body.present? %>
-        <%
-          fallback = t_fallback('contact.access_and_opening_times')
-          lang = fallback if fallback && fallback != I18n.locale
-        %>
-        <%= link_to t('contact.access_and_opening_times'), [contact.worldwide_organisation, contact], class: "url govuk-link", lang: lang %>
+        </div>
       <% end %>
     </div>
-  </div>
+
+    <% if contact.is_a?(WorldwideOffice) && contact.access_and_opening_times_body.present? %>
+      <%
+        fallback = t_fallback('contact.access_and_opening_times')
+        lang = fallback if fallback && fallback != I18n.locale
+      %>
+      <%= link_to t('contact.access_and_opening_times'), [contact.worldwide_organisation, contact], class: "govuk-link", lang: lang %>
```

Comment:
I feel like [`<time>`](http://whitehall-frontend.dev.gov.uk/world/organisations/british-embassy-kabul) could be used on this section for the opening hours. 
However this is probably true everywhere on the site...

Capturing this for later.

Reply:
Something worth noting is that this content isn't the literal opening times but the link to opening times so the time tag wouldn't be appropriate here. All the same I'd forgotten about this tag so cheers for flagging!